documentation/slate/source/includes/interface.md

```diff
@@ -126,3 +126,29 @@ That means :
 * where you can find the details of your object ( **"id": "edc46f3a-ad3d-11e4-a5e1-005056a44da2"** ).
 
 
+
+Objects order
+------------
+
+Unless specified otherwise objects are not sorted and stability of object's order is not guaranteed.
+
+Examples of sorted objects:
+* journeys in a '/journeys' response
+* '/departures' and '/arrivals'
+* stop_points in '/routes/<id_route>/stop_points'
+* '/places_nearby'
+* '/places'
+
```

Comment:
should explain the complex sort of "stop_schedules" ;)
and "traffic_reports" \o/